index.bs

```diff
@@ -248,48 +283,126 @@ spec: mediaqueries-4; urlPrefix: https://drafts.csswg.org/mediaqueries-4/#
+              If the user agent is able to encode the media represented by
+              <a for=MediaCapabilitiesInfo>configuration</a> at a pace that
+              allows encoding frames at the same pace as they are sent to the
+              encoder, set <a for=MediaCapabilitiesInfo>smooth</a> to
+              <code>true</code>. Otherwise set it to <code>false</code>.
+            </li>
+            <li>
+              If the user agent is able to encode the media represented by
+              <a for=MediaCapabilitiesInfo>configuration</a> in a power
+              efficient manner, set <a
+              for=MediaCapabilitiesInfo>powerEfficient</a> to <code>true</code>.
+              Otherwise set it to <code>false</code>. The user agent SHOULD NOT
+              take into consideration the current power source in order to
+              determine the encoding power efficiency unless the device's power
+              source has side effects such as enabling different encoding
+              modules.
+            </li>
+          </ol>
+          <li>
+            Return <var>info</var>.
+          </li>
+        </li>
+      </ol>
     </p>
 
     <p>
-      The <dfn for='MediaDecodingAbility' attribute>supported</dfn> attribute
-      MUST return <code>false</code> if the user agent is not able to decode the
-      media represented by <var>configuration</var>. Otherwise, it MUST return
-      <code>true</code>.
+      The <dfn for='MediaCapabilitiesInfo' attribute>supported</dfn> attribute
+      MUST return <a for=MediaCapabilitiesInfo>supported</a>.
     </p>
 
     <p>
-      The <dfn for='MediaDecodingAbility' attribute>smooth</dfn> attribute MUST
-      return whether the user agent will be able to decode the media at a pace
-      that will allow a smooth playback. If {{MediaDecodingAbility/supported}}
-      returns <code>false</code>, the user agent MUST return <code>false</code>.
+      The <dfn for='MediaCapabilitiesInfo' attribute>smooth</dfn> attribute MUST
+      return <a for=MediaCapabilitiesInfo>smooth</a>.
     </p>
 
     <p>
-      The <dfn for='MediaDecodingAbility' attribute>powerEfficient</dfn>
-      attribute MUST return whether the user agent will be able to decode the
-      media in a power efficient manner. If {{MediaDecodingAbility/supported}}
-      returns <code>false</code>, the user agent MUST return <code>false</code>.
-      The user agent SHOULD not take into consideration the current power source
-      in order to determine the decoding power efficiency unless the device's
-      power source has side effects such as enabling different decoding modules.
+      The <dfn for='MediaCapabilitiesInfo' attribute>powerEfficient</dfn>
+      attribute MUST return <a for=MediaCapabilitiesInfo>powerEfficient</a>.
     </p>
 
     <p class='note'>
-      Authors can use {{MediaDecodingAbility/powerEfficient}} in concordance
+      Authors can use {{MediaCapabilitiesInfo/powerEfficient}} in concordance
```

Comment:
Maybe here s/MediaCapabilitiesInfo/MediaDecodingConfiguration/ since this paragraph speaks about decoding?